src/entity/posts/FreeformPost.ts

```diff
@@ -2,7 +2,7 @@ import { ChildEntity, Column } from 'typeorm';
 import { Post, PostType } from './Post';
 
 // Minimun content length required for new posts to trigger content-requested
-export const FREEFORM_POST_MINIMUM_CONTENT_LENGTH = 50;
+export const FREEFORM_POST_MINIMUM_CONTENT_LENGTH = 5;
```

Comment:
@capJavert should we change this to use GB so we can adjust it on the fly?

Reply:
Yeah, could be good